index.html

```diff
@@ -36,5 +36,135 @@ <h3>LET'S CONNECT</h3>
+            </ul>  
+        </div>
+            <button type="submit" class="project-button">See Project</button>
+        </div>
+</div>        
+<div class="grid-container3"> 
+            <div class="grid-item1"><img src="Snapshoot Portfolio(3).png" alt="portfolio"></div>
+            <div class="grid-item2">
+        <h2>Tonic</h2>
+            <ul >
+                <li style="list-style: none;" id="canopy">CANOPY</li>
+                <li>Back End Dev</li>
+                <li>2015</li>
+            </ul>
+            <p>A daily selection of privately<br>
+                 personalized reads; no accounts or<br>
+                  sign-ups required.</p>
+                  <ul style="list-style: none;" class="language-button">
+                    <button type="submit"><li>html</li></button> 
+                  <button type="submit"><li>css</li></button>
+                    <button type="submit"><li>javascript</li></button>
+                 </ul>  
+            <button type="submit" class="project-button">See Project</button>
+         </div>
+</div>         
+<div class="grid-container4"> 
+           <div class="grid-item1"><img src="Snapshoot Portfolio(4).png" alt="portfolio"></div>
+                 <div class="grid-item2"><h2>Multi-Post <br>Stories</h2>
+            <ul>
+                <li style="list-style: none;" id="canopy">CANOPY</li>
+                <li>Back End Dev</li>
+                <li>2015</li>
+            </ul>
+            <p>A daily selection of privately<br>
+                 personalized reads; no accounts or<br>
+                  sign-ups required.</p>
+                  <ul style="list-style: none;" class="language-button">
+                    <button type="submit"><li>html</li></button> 
+                  <button type="submit"><li>css</li></button>
+                    <button type="submit"><li>javascript</li></button>
+                 </ul>  
+            <button type="submit" class="project-button">See Project</button> 
+    </div>
+</div>    
+    <aside>
+        <h1>About <br>Myself</h1>
+    <p>I’m a software developer! I can help you <br>
+        build a product , feature or website Look <br>
+        through some of my work and <br>
+        experience! If you like what you see and <br>
+        have a project you need coded, don’t <br>hestiate to contact me.</p>
+          <h3>LET'S CONNECT</h3>
+        <ul style="list-style: none;" class="icon"> 
+            <a href="#"><li><img src="./Vector (1).png" alt="github"></li></a>
+            <a style="text-decoration: none;" href="#"><li>
+                <img src="./Vector (4).png" alt="lindedin">
+                <img src="./Vector (5).png" alt="">
+            </li></a>
+            <a href="#"><li><img src="./Vector (2).png" alt="victory"></li></a>
+            <a href="#"><li><img src="./Vector.png" alt="twitter"></li></a>
+            <a href="#"><li><img src="./Vector (3).png" alt="M"></li></a>
+        </ul>
```

Comment:
- Good job using a `ul` tag to wrap your elements here 👍 . But for semantic purposes and better accessibility of your website, please wrap `img` in the `a` tag and wrap the `a` tag in the `li` tag. Please fix this for all icons. 🙏